### PR TITLE
create Header component, and fix bug for authorization when creating job

### PR DESCRIPTION
**Pull request by nickbouldien**
fix authorization issue not letting user create a job when logged in

create Header component to show the user's email when logged in, let them log in/out, go to home, etc

### CODE REVIEW

File: dreamjob_backend/app.js [hunk 1]
```diff
@@ -15,6 +15,9 @@ app.use(express.static('public'))
 app.use(bodyParser.json())
 
 const authorization = function(request, response, next){
+  debugger;
```
Comment (orangewolf) [inline dreamjob_backend/app.js]: please remove these console/debug statements

File: dreamjob_backend/app.js [hunk 2]
```diff
@@ -92,7 +95,8 @@ app.post('/update_job_details/:id', function (request, response){
   })
 })
 
-app.post('/create_job', function (request, response){
+app.post('/create_job', authorization, function (request, response){
+  debugger;
```
Comment (orangewolf) [inline dreamjob_backend/app.js]: 🐛

File: dreamjob_frontend/src/actions/actions.js [hunk 3]
```diff
@@ -161,6 +167,7 @@ export function createJob(attributes){
     body: JSON.stringify(attributes)
   }
   fetch('http://localhost:4000/create_job', params).then((response)=>{
+    debugger;
```
Comment (orangewolf) [inline dreamjob_frontend/src/actions/actions.js]: 🐛

File: dreamjob_frontend/src/stores/UserStore.js [hunk 4]
```diff
@@ -31,7 +32,11 @@ class UserStore extends EventEmitter{
 
   setUserFromLocal(){
     let token = localStorage.getItem('authToken')
+    console.log(token);
```
Comment (orangewolf) [inline dreamjob_frontend/src/stores/UserStore.js]: 🐛

**Review by orangewolf — CHANGES_REQUESTED** [review]

File: dreamjob_backend/app.js (hunk 1 above)
Comment (nickbouldien) [reply dreamjob_backend/app.js]: removed

File: dreamjob_backend/app.js (hunk 2 above)
Comment (nickbouldien) [reply dreamjob_backend/app.js]: we want the authorization in there

**Review by nickbouldien — COMMENTED** [review]
removed

**Review by nickbouldien — COMMENTED** [review]
removed debugs/logs

we want authorization in create_job route

**Pull request merged**